config/ext/rspec.yml

```diff
@@ -762,10 +777,22 @@ RSpec/ReceiveNever:
 
 RSpec/RedundantAround:
   Description: Remove redundant `around` hook.
-  Enabled: pending
+  Enabled: true
   VersionAdded: '2.19'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/RedundantAround
 
+RSpec/RedundantPredicateMatcher:
+  Description: Checks for redundant predicate matcher.
+  Enabled: true
+  VersionAdded: '2.26'
+  Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/RedundantPredicateMatcher
+
+RSpec/RemoveConst:
```

Comment:
https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecredundantpredicatematcher

Hab ich noch nie gesehen. Mir fällt nicht ein, wozu man das brauchen könnte `remove_const` unbedingt im Test braucht. Von mir aus :+1:

Reply:
I don't think I've seen this either, so fine by me.